examples/01_example_wsiread.ipynb

```diff
@@ -21,20 +21,40 @@
     "id": "p3l08vukKAWo"
```

Comment:
<p><em>I'm sorry that I have been so slow on the tissuemask notebook. While I was working on that notebook, I decided that the following documentation (with appropriate deletions in the code cell) would be less puzzling to a naive user:</em></p><p> </p><p>This cell should be skipped, unless you are using Colab or Kaggle. If you are using either Colab or Kaggle, this cell prepares the Python environment for<code>tiatoolbox</code>, which may give rise to harmless error messages: please ignore them.</p><p> </p><p>Outside Colab or Kaggle, the notebook expects&nbsp;<code>tiatoolbox</code>to already be installed. (See the instructions in&nbsp;<a href="https://github.com/TIA-Lab/tiatoolbox/blob/master/README.md#install-python-package" target="_blank">README</a>.)</p><p><em>If we decide to do this, then the next code cell should be deleted, which is an improvement on my earlier approach.</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TIA-Lab/tiatoolbox/pull/132/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Addressed in this commit https://github.com/TIA-Lab/tiatoolbox/pull/132/commits/0a7e3eef1bcb98b1137180859fd0cdeb8769d9e9